tests/basic/volume_ops_test.go

```diff
@@ -2843,6 +2843,111 @@ var _ = Describe("{NFSProxyVolumeValidation}", func() {
 	})
 })
 
+var _ = Describe("{SharedVolFuseTest}", func() {
+	/*
+
+				https://portworx.atlassian.net/browse/PWX-35639
+			   https://portworx.atlassian.net/browse/PTX-21805
+
+
+		       	1. Deploy apps  with mix of sv4 service and sv4  using k8s
+		  		2.Get the list of coordinator nodes have sv4 volume attached
+				3.Stop/Start PX on the coordinator node
+				4.Check the failover has happened i.e. we  have new  coordinator node assigned after step 3
+		   		5.Repeat this in a loop for 10 iterations
+
+
+
+	*/
+	var contexts []*scheduler.Context
+	JustBeforeEach(func() {
+		StartTorpedoTest("SharedVolFuseTest", "Validate PX operations after sharedv4 and sharedv4 svc volumes  failover multiple times", nil, 0)
+	})
+
+	It("schedule sharedv4 and sharedv4_svc volumes and perform failover of the coordinator node", func() {
+
+		stepLog = "create sharedv4 and sharedv4_svc apps "
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			appList := Inst().AppList
+
+			defer func() {
+				Inst().AppList = appList
+			}()
+
+			Inst().AppList = []string{"vdbench-sv4-svc", "fio-sharedv4"}
+			contexts = make([]*scheduler.Context, 0)
+
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				contexts = append(contexts, ScheduleApplications(fmt.Sprintf("pxfusetest-%d", i))...)
+			}
+
+			for _, ctx := range contexts {
+				log.InfoD("Validating application [%s]", ctx.App.Key)
+				ValidateContext(ctx)
+			}
+		})
+
+		stepLog = "restart PX on coordinator nodes"
```

Comment:
IMO, this test should be time based, until time expires, test should run in a loop. Can you modify the test accordingly?

Reply:
Also, bug QA notes say it is on different kernel versions, make sure we are running this test on different kernel versions
< 5.13
>= 5.13 and < 5.15.25
>= 5.15.25

Reply:
> Also, bug QA notes say it is on different kernel versions, make sure we are running this test on different kernel versions < 5.13
> 
> > = 5.13 and < 5.15.25
> > = 5.15.25

yes, running this test in different platforms should cover it.

Reply:
> IMO, this test should be time based, until time expires, test should run in a loop. Can you modify the test accordingly?

spoke to @gpal1 and he mentioned , he is able to reproduce the issue by restarting all nodes hence changed the logic to reboot all nodes for 10 iterations each